app/views/govuk_publishing_components/components/docs/govspeak.yml

```diff
@@ -181,6 +181,7 @@ examples:
           </tbody>
         </table>
   charts:
+    description: "[We recommend a limit of four categories as best practice for basic data visualisations](https://gss.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/#section-5)."
```

Comment:
I think we need a stronger message here that tells people it will fail after 5 colours. Maybe:

We recommend a limit of four categories as best practice for basic data visualisations and only a maximum of five will render correctly.

Reply:
Yep that should be made clearer, i've updated now to

> The Government Statistical Service (GSS) guidance recommends [a limit of four categories as best practice for basic data visualisations](https://gss.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/#section-5).

Note that charts which have five categories, [chart with colours](http://govuk-publishing-components.dev.gov.uk/component-guide/govspeak/chart_with_colours/preview), for example, will display the fifth bar in the default colour of #0b0c0c and will still meet the colour contrast requirements for adjacent colours. Charts that have six or more categories will display additional bars in the default colour and will not meet colour contrast requirements for adjacent colours.